weave/trace_server/sqlite_trace_server.py

```diff
@@ -91,7 +91,9 @@ def setup_tables(self) -> None:
                 parent_id TEXT,
                 op_name TEXT,
                 started_at TEXT,
+                started_at_micros TEXT,
                 ended_at TEXT,
+                ended_at_micros TEXT,
```

Comment:
since the data is exactly the same here, i don't think we need any new fields for sqlite?